src/components/Feed/Feed.tsx

```diff
@@ -2,24 +2,28 @@ import React from 'react';
 import Card from '../Card/Card';
 import { Article } from '../App/App';
 import './Feed.css';
+import { Link } from 'react-router-dom';
+import './Feed.css';
 
 interface PropsFeed {
   articles: Article[]
 }
 
 const Feed: React.FC<PropsFeed> = ({ articles }) => {
   return (
-    <div className="articles-container">
-      <section className="articles-display">
-        {articles.map(article =>
-          <Card
-          title={article.title}
-          image={article.multimedia[0].url}
-          key={article.title}
-          />
-        )}
-      </section>
-    </div>
+    <Link to="/WellNewsFeed">
```

Comment:
This Link directs to same page where the user is on this page. We will need a Link for each of the Card components once we have them go to the single article view, but just curious what the functionality of this Link is.